### PR TITLE
CEDS-2785 Redirect start page to the choice page

### PR DESCRIPTION
**Pull request by glidester**
This work is prep for removing the start page completely
from the user journey (as it will be replaced by the
Gov.uk page).

### CODE REVIEW

File: app/controllers/StartController.scala [hunk 1]
```diff
@@ -25,6 +25,6 @@ import views.html.start_page
 class StartController @Inject()(mcc: MessagesControllerComponents, startPage: start_page) extends FrontendController(mcc) with I18nSupport {
```
Comment (udinobi) [inline app/controllers/StartController.scala:25]: Do we still need the start_page view? Will it maybe removed in future?

Comment (glidester) [reply app/controllers/StartController.scala:25]: Yes, next ticket 2965 we will remove all links to the start page and remove the controller and view then.

Comment (udinobi) [reply app/controllers/StartController.scala:25]: OK, but do we still need to inject `startPage`? It's not used anywhere.

**Review by udinobi — APPROVED** [review]

File: app/controllers/StartController.scala (hunk 1 above)
Comment (glidester) [reply app/controllers/StartController.scala]: Just ended up pulling the whole start_page view in the end :+1:

**Pull request merged**